docs/getting-started/quick-start.md

```diff
@@ -16,7 +16,7 @@ You have a few methods to choose from:
 * [Using CKEditor&nbsp;5 Builder](#using-ckeditor-5-builder) for the smoothest setup with live preview and multiple integration options.
 * [Using npm](#installing-ckeditor-5-using-npm) where you use a JavaScript package and build the editor with a bundler.
 * [Using CDN](#installing-ckeditor-5-from-cdn), where you use our cloud-distributed CDN in a no-build setup.
-* [Using a provided JavaScript file](#installing-ckeditor-5-from-a-file) where you download the ready-to-run file and copy them to your project.
+* [Using a provided JavaScript file](#installing-ckeditor-5-from-a-zip-file) where you download the ready-to-run file and copy them to your project.
```

Comment:
Maybe the title of this link should also mention ZIP?

> Using a ZIP file

There st of the description ("where you download...") describes the rest.

Reply:
Yes, it should. I'm changing things currently, and it is in my changes.